src/Contexts/ChainbridgeContext.tsx

```diff
@@ -122,6 +123,8 @@ const ChainbridgeProvider = ({
     setAddress(undefined);
     setTransferTxHash(undefined);
     setHomeTransferTxHash(undefined);
+    const disconnected = !isReady && transactionStatus === "Transfer Completed";
```

Comment:
Very bad naming. Condition looks close to `transfer completed` state, no?

Reply:
It means that the wallet disconnected after the transfer was completed.

Reply:
I see.
Do we really need to check `transactionStatus === *`?

Reply:
Fixed